test/parallel/test-https-agent-disable-session-reuse.js

```diff
@@ -1,17 +1,17 @@
 'use strict';
 const common = require('../common');
+const fixtures = require('../common/fixtures');
 if (!common.hasCrypto)
   common.skip('missing crypto');
 
 const assert = require('assert');
 const https = require('https');
-const fs = require('fs');
 
 const TOTAL_REQS = 2;
 
 const options = {
-  key: fs.readFileSync(`${common.fixturesDir}/keys/agent1-key.pem`),
-  cert: fs.readFileSync(`${common.fixturesDir}/keys/agent1-cert.pem`)
+  key: fixtures.readSync('/keys/agent1-key.pem'),
```

Comment:
We also can use `fixtures.readFixtureKey`.
```diff
+  key: fixtures.readFixtureKey('agent1-key.pem'),
-  key: fixtures.readSync('/keys/agent1-key.pem'),
```

Reply:
in the fixtures.js the readFixturesKey is exported as readKey. I have updated the branch